imap_processing/ultra/l1c/ultra_l1c_pset_bins.py

```diff
@@ -247,8 +247,36 @@ def get_background_rates(
     -----
     This is a placeholder.
     """
-    n_pix = hp.nside2npix(nside)
-    return np.zeros(n_pix)
+    npix = hp.nside2npix(nside)
+    _, energy_midpoints, _ = build_energy_bins()
+    background = np.zeros((len(energy_midpoints), npix))
+    return background
+
+
+def get_helio_background_rates(
+    nside: int = 128,
+) -> NDArray:
+    """
+    Calculate background rates.
+
+    Parameters
+    ----------
+    nside : int, optional
+        The nside parameter of the Healpix tessellation (default is 128).
+
+    Returns
+    -------
+    background_rates : np.ndarray
+        Array of background rates.
+
+    Notes
+    -----
+    This is a placeholder.
```

Comment:
Do you want to add `"TODO: "` to any comment on a placeholder to make it easier to track? There's at least one other above this.